lib/screens/user_registration/user_registration.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:teesco/screens/login/widgets/password_field.dart';
+import 'package:teesco/screens/user_registration/widgets/email_input.dart';
+import 'package:teesco/screens/user_registration/widgets/institute_name_input.dart';
+import 'package:teesco/screens/user_registration/widgets/phone_number_field.dart';
+import 'package:teesco/screens/user_registration/widgets/confirm_password_field.dart';
+
+class UserRegistration extends StatelessWidget {
+  TextEditingController userNameController;
+  TextEditingController emailController;
+  TextEditingController passwordController;
+  TextEditingController confirmPasswordController;
+  TextEditingController phoneNumberController;
+  TextEditingController instituteNameController;
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Form(
+            child: SafeArea(
+      child: SingleChildScrollView(
```

Comment:
why the scroll view?

Reply:
The fields at the bottom would not be accessible without any ScrollView widget